src/lib/python/chess/board.py

```diff
@@ -234,12 +239,21 @@ def draw_move_with_preview(
 
 
 def get_move_audio(move_san: str):
-    move_audio_clip_name = "move"
-    if move_san.endswith("#"):
-        move_audio_clip_name = "checkmate"
-    elif move_san.endswith("+"):
-        move_audio_clip_name = "check"
-    elif "x" in move_san:
-        move_audio_clip_name = "capture"
-
-    return editor.AudioFileClip(f"./src/resources/chess/{move_audio_clip_name}.mp3")
\ No newline at end of file
+    audio_mapping = {
+        "#": ["check", "checkmate"],
+        "+": ["check"],
+        "x": ["capture"]
+    }
+
+    move_audio_clip_names = []
+    for key, clips in audio_mapping.items():
+        if key in move_san:
+            move_audio_clip_names.extend(clips)
+            break
+    else:
+        move_audio_clip_names.append("move")
+
+    audio_clips = [editor.AudioFileClip(
+        f"./src/resources/chess/{clip}.mp3") for clip in move_audio_clip_names]
+
+    return editor.concatenate_audioclips(audio_clips)
```

Comment:
thoughts: have you tested this? does `concatenate_audioclips` put the audio clips over each other? or does it just play one after the other? the checkmate and game end sound need to play at the same time.

Reply:
I hadn't realized that, i think it's a good way do something like this:
```py
if move_san.endswith("#"):
        check_clip = editor.AudioFileClip(f"./src/resources/chess/check.mp3")
        checkmate_clip = editor.AudioFileClip(f"./src/resources/chess/checkmate.mp3")
        combined_clip = editor.CompositeAudioClip([check_clip, checkmate_clip])
        audio_clips = [combined_clip]
```

(ignore the voice in the clips, it's an possible update I'm doing)
Which is better?
before:

https://github.com/WintrCat/autoshorts/assets/168477146/b9b359b4-a879-4426-8a63-c00d873edadd

after:

https://github.com/WintrCat/autoshorts/assets/168477146/1925fd07-3d1b-4801-9877-52203e8811ff

Reply:
The "after" one is better - this is the configuration already present on Chess.com

Reply:
> The "after" one is better - this is the configuration already present on Chess.com

Okay, i will find a good way to implement this condition without unnecessary code